conf/solr/crawldb/conf/managed-schema

```diff
@@ -116,20 +116,48 @@
    <!-- Crawl DB Static Fields -->
    <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
    <field name="url" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
-   <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
    <field name="group" type="string" indexed="true" stored="true" required="true"  multiValued="false" docValues="true" /> 
-   <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" />
-   <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" />
-   <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
    <field name="score" type="double" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" />
 
+   <!-- Deprecated Fields -->
+   <!-- <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" /> -->
+   <!-- <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
+
+   <!-- Proposed Sparkler Schema -->
+   <field name="fetch_timestamp" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="content_type" type="string" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="crawler" type="string" indexed="true" stored="true" multiValued="false" default="sparkler"/> 
+   <field name="crawl_id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
+   <field name="raw_content" type="text_general" indexed="true" stored="true" multiValued="false" />
```

Comment:
@karanjeets trying to understand why snake case is preferred instead of camel case?
I would prefer camelCase here because -
whether we transfer in xml or json, these fields gets converted to characters. Thus Increase in bytes count since `_` is an additional char added everywhere!

Is it because CDR schema is defined this way and we want to stick with it?